src/data/markdown/docs/02 javascript api/11 k6-execution.md

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
This shouldn't be number 11, it should be alphabetically ordered after `k6/encoding`.

Also, this is wrong: 
![Screenshot 2021-09-08 at 11-31-13 k6 execution](https://user-images.githubusercontent.com/3336773/132475106-e79cae63-681a-42ca-8b4e-5cc036354585.png)


`k6/execution` doesn't have an `Execution` object inside of it, rather the `default` export is an object with some properties, right @MStoykov and @imiric. Not sure how to actually format that with the current docs format though, @ppcano any ideas?

Reply:
aaaah ... yeah, it is just the default import (or the only thing `require` will return) so I guess it could just be one page with a note on top ?